server/src/test/java/org/candlepin/service/impl/stub/StubEntitlementCertServiceAdapter.java

```diff
@@ -87,4 +91,31 @@ public EntitlementCertificate generateUeberCert(Entitlement entitlement,
         return generateEntitlementCert(entitlement, product);
     }
 
+    @Override
+    public Map<String, EntitlementCertificate> generateEntitlementCerts(Consumer consumer,
+            Map<String, Entitlement> entitlements, Map<String, Product> products)
+        throws GeneralSecurityException, IOException {
+
+        Map<String, EntitlementCertificate> result = new HashMap<String, EntitlementCertificate>();
+        for (Entry<String, Entitlement> entry : entitlements.entrySet()) {
+            EntitlementCertificate cert = generateEntitlementCert(entry.getValue(),
+                    products.get(entry.getKey()));
+            result.put(entry.getKey(), cert);
+        }
+        return result;
+    }
+
+    @Override
+    public Map<String, EntitlementCertificate> generateUeberCerts(Consumer consumer,
+            Map<String, Entitlement> entitlements, Map<String, Product> products)
+        throws GeneralSecurityException, IOException {
```

Comment:
> if(you mean to suggest that we change the interface too) {

Sorry, I misread that section of code.  I thought it was a test since that's mostly what I reviewed yesterday.  I totally agree that actual deployed code should throw the appropriate exception, but I usually have tests just throw Exception.